backend/controllers/prisonerProfile/prisonerLocationHistory.js

```diff
@@ -12,7 +12,43 @@ const {
   app: { notmEndpointUrl: dpsUrl },
 } = require('../../config')
 
-module.exports = ({ prisonApi, logError }) => async (req, res) => {
+const fetchStaffName = (context, staffId, prisonApi) =>
+  prisonApi.getStaffDetails(context, staffId).then(staff => formatName(staff.firstName, staff.lastName))
+
+const fetchReasonDescription = (context, assignmentReasonCode, caseNotesApi) => {
+  if (assignmentReasonCode) {
```

Comment:
This seems strange. Sometimes it returns a promise and other times a string. Might be worth doing the checkout outside of this method.

Reply:
sure will revert it to how it was before